LibraBFT/Impl/Handle.agda

```diff
@@ -50,6 +50,7 @@ postulate -- TODO-2: define GenesisInfo to match implementation and write these
 initSR : SafetyRules
 initSR =
   let sd = fakeRM ^∙ lSafetyRules
```

Comment:
no big deal, but this variable should be named `sr` (i.e., `SafetyRules`).

Reply:
@haroldcarr I have renamed `sd` to `sr`